src/main/java/beans/StatusResponseBean.java

```diff
@@ -0,0 +1,41 @@
+package beans;
+
+/**
+ * A generic status response that returns either "ok" or "error" with a message
+ * if it has errored.
+ */
+public class StatusResponseBean {
+    public static final String OK = "ok";
+    public static final String ERROR = "error";
+    public static final String GENERIC_ERROR = "There was an error processing the request";
+
+    private String status;
+    private String message;
+
+    public StatusResponseBean() {}
+
+    public StatusResponseBean(Boolean success) {
```

Comment:
using `Boolean` instead of `boolean` is usually unnecessary (only really need the wrapper class when you have operations that require an object)